### PR TITLE
Blog Post for Silver Sponsor Adobe

### PR DESCRIPTION
**Pull request by HimanshuAwasthi95**

### CODE REVIEW

File: content/blog/Adobe-silver-sponsor.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Title: Announcing Adobe as Silver Sponsor
+Date: 2017-10-18 10:32
+Category: 2017
+Slug: silver-sponsor-Adobe
+Summary: Silver Sponsor Adobe
+
+
+<img src="https://in.pycon.org/2017/images/sponsor/adobe.jpg">
+
+**[Adobe Systems Incorporated](http://www.adobe.com/in/)** is an American multinational computer software company. The company is headquartered in San Jose, California, United States. Adobe's focus has always been the creation of multimedia and creativity software products, with foray of rich Internet application software. Adobe tools enables to take productivity to the next level while preserving the creative vision.
```
Comment (realslimshanky) [inline content/blog/Adobe-silver-sponsor.md]: Please make the following corrections
foray -> the foray
tools enables -> tools enable

File: content/blog/Adobe-silver-sponsor.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Category: 2017
+Slug: silver-sponsor-Adobe
+Summary: Silver Sponsor Adobe
+
+
+<img src="https://in.pycon.org/2017/images/sponsor/adobe.jpg">
+
+**[Adobe Systems Incorporated](http://www.adobe.com/in/)** is an American multinational computer software company. The company is headquartered in San Jose, California, United States. Adobe's focus has always been the creation of multimedia and creativity software products, with foray of rich Internet application software. Adobe tools enables to take productivity to the next level while preserving the creative vision. 
+
+Adobe offers products and services that are used by professionals, marketers, knowledge workers, application developers and enterprises for creating, managing, delivering, measuring, optimizing and engaging with compelling content and experiences. It is best known development for [Photoshop](http://www.photoshop.com/), an image editing software, [Acrobat Reader](https://get.adobe.com/reader/otherversions/), the Portable Document Format (PDF) and [Adobe Creative Suite](https://en.wikipedia.org/wiki/Adobe_Creative_Suite), as well as its successor [Adobe Creative Cloud](http://www.adobe.com/in/creativecloud.html).
```
Comment (realslimshanky) [inline content/blog/Adobe-silver-sponsor.md]: Please make the following corrections
best known -> best-known

**Comment by HimanshuAwasthi95** [comment]
Done Brother

**Comment by realslimshanky** [comment]
Awesome @HimanshuAwasthi95 
Thanks for your contribution. :pizza:

**Pull request merged**